# message for multi uav communication

x_vio_ros/FeatureMsg[] features
uint8[] descriptor
uint64 id
float64[] landmark
